src/models/task.js

```diff
@@ -366,7 +366,9 @@ export default class Task {
 	}
 
 	set note(note) {
-		this.vtodo.updatePropertyWithValue('description', note)
+		// To avoid inconsistent parameters, delete complete property, then recreate
```

Comment:
```suggestion
		// To avoid inconsistent property parameters (bug #3863), delete the property, then recreate
```

Reply:
Done! (Add: issue is in nextcloud/calendar)